content/boot-menu.md

```diff
@@ -16,15 +16,15 @@ section: software
 tableOfContents: true
 ---
 
-To enter BIOS for most System76 laptops, hold the <kbd>F2</kbd> key immediately after turning your computer on.  For the boot menu, hold down the <kbd>F7</kbd> key.  For older laptops, hold the <kbd>F1</kbd> key.
-
-| Firmware               | BIOS key | Boot Menu key                    |
-|:----------------------:|:--------:|:--------------------------------:|
-| Laptop - Open Firmware | ESC      | ESC(select one time boot option) |
-| Laptop - Proprietary   | F2       | F7                               |
-| Older Laptops          | Depends on the system | F1                  |
-| Thelio                 | Del      |  F8 or F12                       |
-| Meerkat                | F2       | F10                              |
+The BIOS or boot menu can be accessed by holding specific keys down during boot. Refer to the table below for the keys on each model:
+
+| Model                                                       | BIOS key | Boot Menu key                     |
+|:-----------------------------------------------------------:|:--------:|:---------------------------------:|
+| Laptops:<br/>All Open Firmware models<br/>Pangolin (pang12) | ESC      | ESC (select one-time boot option) |
```

Comment:
I wonder if the `(select one-time boot option)` note should be tweaked to account for pang12's boot menu:

![IMG_20230320_155402_417](https://user-images.githubusercontent.com/13512097/226474276-d9d0e4d1-cc5b-4c3b-acd5-24d85f015424.jpg)

I'm thinking maybe something like, "(select one-time boot option in open firmware, or use the boot override menu in "save & exit" on pang12)," but it's awfully wordy.

Reply:
Is this better?

![image](https://user-images.githubusercontent.com/7199422/226475364-abfb4ae1-fecf-46bf-bcd1-85f3a5349d86.png)

Reply:
I guess we might want the page that it's on for proprietary.

Reply:
Here's another option:

![image](https://user-images.githubusercontent.com/7199422/226475868-4a650e9e-7b8b-461b-89a5-0314f787e675.png)

Is that clear enough?